src/views/AmalgamationSelection.vue

```diff
@@ -231,7 +265,11 @@ export default class AmalgamationSelection extends Vue {
           nameRequest: {
             legalType
           },
-          type
+          type,
+          contactPoint: {
```

Comment:
To populate email and phone. Otherwise, shows 'Not available'

![image](https://github.com/bcgov/business-filings-ui/assets/116035339/8d48be0e-28a5-48e2-98c2-ad8897fd94b8)

Reply:
It's not "required" because it's prompted for in the amalgamation (step 2).

Please confirm with Yui whether we should pre-populate business contact info. I think it's OK either way. If he says yes then ask if we should do that for reg amalgamations, too.

Reply:
I will check with him

Reply:
@severinbeauvais 
From Yui: They should be autofilled for short form's holding or primary business. Note that the flag for holding/primary can be changed in step 1 by the user and that means the content of step 2 needs to update when the user makes the flag change.

based on this I am updating in filings-ui. Need any update in create-ui?

Reply:
Yes, please add to ticket 19037 that Registered Office Contact Information also needs to be updated when the draft is restored, and if the holding/primary company changes. << FYI, @vysakh-menon-aot 

Please also verify with Yui that the ROCI remains editable in the amalgamation filing, even though it is prepopulated byt the holding/primary company. (This makes it different from the Office Addresses, Directors, and Share Structure.)

Reply:
For first: I will add to the ticket #19037.
@yuisotozaki could you please verify second?

Reply:
Yes - ROCI remains editable post auto-populate.

Reply:
Thanks, Yui.

Ketaki, please resolve this conversation when you've updated that other ticket.

Reply:
Updated #[19037](https://app.zenhub.com/workspaces/entities---olga-65af15f59e89f5043c2911f7/issues/gh/bcgov/entity/19037)